pom.xml

```diff
@@ -1399,5 +1407,19 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
```

Comment:
Not a particular burden. It's just arbitrary. Why profiles for these particular combination of groups/excludes? Why not others? When we add more categories (which we almost certainly will), how much sense will it make to preserve a single distinct combination, but others?

I just don't see much value in establishing these particular presets, and it will be confusing as more categories are added why this particular subset of possible presets are added as convenience profiles for everybody.

If there's a good reason why these particular presets are useful, above all other potential future combinations of categories, for now and in the future, then I'd say we'd have a strong reason to pre-define them. Otherwise, I think they're better omitted, and not singled out in the docs.